### DISCUSSION TITLE
Remove resolveState() call from "Persisting State" section in docs?

### DISCUSSION THREAD

**Discussion by aerhard** [statelyai/xstate#2811] [General] [answered]
The "Persisting State" section of the docs lists the following steps to restore a serialized actor (https://github.com/statelyai/xstate/blob/main/docs/guides/states.md?plain=1#L283-L307):

```
const previousState = State.create(stateDefinition);
const resolvedState = myMachine.resolveState(previousState);
const service = interpret(myMachine).start(resolvedState);
```

Since the interpreter's `start()` method calls `resolveState()` when its `initialState` argument is a `State` instance (https://github.com/statelyai/xstate/blob/main/packages/core/src/interpreter.ts#L472-L483), the `resolveState()` call in the docs seems redundant. Should it be removed?

**Comment by davidkpiano** [comment, marked as answer]
Yes, it can be removed. Want to make a PR?

**Reply by aerhard** [reply]
Yeah, I'll do!
